ordia/app/templates/search.html

```diff
@@ -1,5 +1,18 @@
 {% extends "base.html" %}
 
+{% block scripts %}
+{{super()}}
+
+<script type="text/javascript" src="{{ url_for('static', filename='lexemeforms.js') }}"></script>
+<script type="text/javascript">
+ $(document).ready(function() {
+     lexemeTemplateSelection("{{ lang }}", "{{ query }}", "#lexeme-templates");
```

Comment:
The escape is not proper.

Reply:
https://stackoverflow.com/questions/12339806/escape-strings-for-javascript-using-jinja2